src/Builders/PaymentMethodBuilder.php

```diff
@@ -167,6 +167,36 @@ public function get(): Collection
         }
     }
 
+    /**
+     * Set as default payment and/or of current subscriptions.
+     *
+     * @param  array<string, bool>  $payload
+     * @return bool
+     */
+    public function setAsDefault(array $payload = []): bool
```

Comment:
It's better if you simply accept the `bool $update` here and build the payload within the method.